frontend/routes/package/(_components)/PackageHeader.tsx

```diff
@@ -5,6 +5,7 @@ import { RuntimeCompatIndicator } from "../../../components/RuntimeCompatIndicat
 import { getScoreTextColorClass } from "../../../utils/score_ring_color.ts";
 import {
   TbAlertTriangleFilled,
+  TbBug,
```

Comment:
```suggestion
  TbFlag,
```

Usage of TbFlag to more related with the action of "reporting"